analyses/ssgsea-hallmark/02-analyze-plot-ssgsea-res.R

```diff
@@ -6,54 +6,55 @@
 
 
 #Call libraries
-library("tidyverse");
-library("pheatmap");
-library("RColorBrewer");
+library("tidyverse")
+library("pheatmap")
+library("RColorBrewer")
 library("viridis")
 
 #Read in data
-clinData <- read.delim("../../data/pbta-histologies.tsv", stringsAsFactors=F);
+clinData <- read.delim("../../data/pbta-histologies.tsv", stringsAsFactors=F)
 geneSetExpMat <- readRDS("../../scratch/GeneSetExpressionMatrix.RDS")
 
 #Choose Clinical Samples corresponding to samples in Gene Set Matrix
 clinData <- clinData[clinData[,"experimental_strategy"]=="RNA-Seq",]
 rownames(clinData) <- clinData[,"Kids_First_Biospecimen_ID"]
 clinData <- clinData[colnames(geneSetExpMat),]
 
-#Analyse Pathways statistically correlated with clinical variables
 getVarNames <- function(x)
```

Comment:
Documentation about what this function aims to do is missing.

Reply:
Updated in code ... will push when all other changes have been made
```
Function to split pathway names from TukeyHSD into vector of 2 elements
i.e. Tukey HSD outputs Pathway1-Pathway2 and this will spitout c(Pathway1, Pathway2)
Will be used so that we can tell quickly which pathways are consistently upregulated vs other pathways
```

Reply:
Can you edit this PR description to more precisely describe the steps you're aiming to carry out and provide a scientific rationale for them?

Reply:
Sure,  updated.